version/versions.yaml

```diff
@@ -1,4 +1,6 @@
 - operatorVersion: 1.3.0
   supportedIstioVersions: 1.3.0
   recommendedIstioVersions: 1.3.0
-
+- operatorVersion: 1.4.0
+  supportedIstioVersions: ">=1.3.0,<=1.4"
```

Comment:
We haven't yet finalized whether we'll have forward compatibility in a minor version, so let's keep this to 1.4.0 for now.

Reply:
removed 1.4.0 entry.